test/mockserver/pendo-expectations.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+      "id": "PendoTrackMissingKey",
+      "httpRequest": {
+        "method": "POST",
+        "path": "/data/track",
+        "headers": {
+          "!x-pendo-integration-key": [ ".*" ]
+        }
+      },
+      "httpResponse": {
+        "statusCode": 401,
+        "body": "Missing x-pendo-integration-key"
+      }
+    },
+
+    {
+      "id": "PendoTrackBadContentType",
+      "httpRequest": {
+        "method": "POST",
+        "path": "/data/track",
+        "headers": {
+          "Content-Type": [ "!application/json" ]
+        }
+      },
+      "httpResponse": {
+        "statusCode": 400,
+        "body": "Bad Content-Type"
+      }
+    },
+
+    {
+      "id": "PendoTrack",
+      "httpRequest": {
+        "method": "POST",
+        "path": "/data/track",
+        "body": {
+          "type": "JSON",
+          "json": {
+              "type": "track"
+          },
+          "matchType": "ONLY_MATCHING_FIELDS"
+        }
+      },
```

Comment:
Not sure if intended or not, and it probably doesn't matter... I noticed while testing that if we satisfy the integration key and content-type, but fail to satisfy the json body match, mockserver responds 404.  I deduce that lack of match simply "falls off" the end, causing 404, rather than a 400 (which is how I would expect the real server to behave).

Again, it probably doesn't matter for our purposes but I thought I should share this observation - might help somehow in the future :)